README.md

```diff
@@ -50,14 +50,36 @@ make install
 | `xtensor-fftw` | `xtensor`        | `xtl`   | `fftw`  |
 |----------------|------------------|---------|---------|
 |  master        | >=0.20.9,<0.22   |  ^0.6.9 | ^3.3.8  |
-|  0.2.6         | >=0.20.9,<0.22   |  ^0.6.9 | ^3.3.8  |
+|  0.3.0         | >=0.20.9,<0.22   |  ^0.6.9 | ^3.3.8  |
```

Comment:
This should be updated when releasing only. The new version must be added, and we usually keep the older ones for a while.

Reply:
Good to known.